CMake replaces @SHARED_LIBS_INSTALL_RPATH@ with /usr/lib, causing
abuild to warn about D aports having redundant /usr/lib in rpath.

--- a/ldc/ldc2_install.conf.in
+++ b/ldc/ldc2_install.conf.in
@@ -29,7 +29,7 @@ default:
         "@CMAKE_INSTALL_LIBDIR@",@OPTIONAL_COMPILER_RT_DIR@
     ];
     // default rpath when linking against the shared default libs
-    rpath = "@SHARED_LIBS_INSTALL_RPATH@";
+    rpath = "";
 };
 
 "^wasm(32|64)-":
